client/src/app/payroll/payroll.component.html

```diff
@@ -1 +1,39 @@
-<div class="table-margin"><app-payroll-list></app-payroll-list></div>
+<div class="container">
+  <div class="row">
+    <div class="col-2 d-flex">
+      <span>
+        <h4>Departments:</h4>
+        <ul
+          ngbNav
+          #nav="ngbNav"
+          [(activeId)]="department"
+          class="nav-pills"
+          orientation="vertical"
+        >
+          <li ngbNavItem="Human Resources">
+            <a ngbNavLink>Human Resources</a>
+          </li>
+          <li ngbNavItem="IT">
+            <a ngbNavLink>IT</a>
+          </li>
+          <li ngbNavItem="Sales">
+            <a ngbNavLink>Sales</a>
+          </li>
+          <li ngbNavItem="Building Services">
+            <a ngbNavLink>Building Services</a>
+          </li>
+          <li ngbNavItem="Marketing">
+            <a ngbNavLink>Marketing</a>
+          </li>
+          <li ngbNavItem="Warehouse">
+            <a ngbNavLink>Warehouse</a>
+          </li>
+        </ul>
+      </span>
+      <div [ngbNavOutlet]="nav" class="ml-4"></div>
+    </div>
+    <div class="col-10">
+      <app-payroll-list department="{{ department }}"></app-payroll-list>
```

Comment:
This works, though there is a more common syntax for binding values to component `@Input()`s.

```
<app-payroll-list [department]="department"></app-payroll-list>
```

Here `[department]` binds to the component property (JS) instead of the component attribute (HTML).

The binding syntax `[someProp]` can be assigned any JS expression - in this case the value of the parent component `department` field.

Reply:
One thing you'll often see is when needing to bind a literal string value to something like `@Input name: string`, instead of writing `<a-component [name]="'Sean'"></a-component>`, you can use the HTML attribute and write `<a-component name="Sean"></a-component>`, which lets you avoid the quotes.

HTML attributes require strings (`name="Sean"`) whereas JavaScript properties require expressions (`[name]="'Sean'"`).